terraform-plans/templates/github/snap_release.yaml.tftpl

```diff
@@ -46,3 +46,30 @@ jobs:
           snap: $${{ env.SNAP_FILE }}
           # Comma-separated list of channels to release the snap to.
           release: ${channels}
+
+  notify-on-release-failure:
+    runs-on: ubuntu-latest
+    needs:
+      - check
+      - release
+    if: failure()
+    steps:
+      - name: Comment on commit if release workflow fails
+        uses: actions/github-script@v7
+        with:
+          script: |
+            const failedJobs = $${{ toJSON(needs) }};
+            let failedJobNames = Object.keys(failedJobs).filter(job => failedJobs[job].result === 'failure');
+            await github.rest.repos.createCommitComment({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              commit_sha: context.sha,
+              body: `
+                Release workflow \`$${{ github.workflow }}\` failed!
+                Failed jobs: $${failedJobNames.join(', ') || 'unknown'}
+                Check details: $${{ github.server_url }}/$${{ github.repository }}/actions/runs/${{ github.run_id }}
```

Comment:
```suggestion
                Check details: $${{ github.server_url }}/$${{ github.repository }}/actions/runs/$${{ github.run_id }}
```

Reply:
CI catched this issue. I've fixed, thanks